Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:xenial
+
+USER root
+RUN apt-get update && apt-get install -y \
+    mongodb \
```

Comment:
aww db running in the same container? i guess that's easiest given the current setup...